constants.js

```diff
@@ -1,3 +1,9 @@
 module.exports = Object.freeze({
-  docPagesDirectory: `${__dirname}/src/pages/docs`,
+  docPagesDirectory: `${__dirname}/src/pages/docs/`,
+  versions: [
+    'stable', // has to be the first one.
+    'master',
+    '2.3',
+    '2.2',
```

Comment:
Why not the older branches?

Reply:
This is arbitrary. I'll add the previous ones.